platform/cog-platform-fdo.c

```diff
@@ -1122,56 +1155,78 @@ init_egl (void)
     };
 
     EGLint num_configs;
-    if (! eglChooseConfig(egl_data.display,
-                          config_attribs,
-                          &egl_data.egl_config,
-                          1,
-                          &num_configs)) {
-        assert (!"Error choosing EGL config\n");
+    if (!eglChooseConfig(egl_data.display,
```

Comment:
Missing space here after `eglChooseConfig`

Reply:
Fixed in update, thanks!